spec/unit/mixin/shell_out_spec.rb

```diff
@@ -57,38 +56,38 @@ def env_path
           describe "and environment is an option" do
             it "should not change environment language settings when they are set to nil" do
               options = { environment: { "LC_ALL" => nil, "LANGUAGE" => nil, "LANG" => nil, env_path => nil } }
-              expect(Chef::Mixin::ShellOut).to receive(:shell_out_command).with(cmd, **options).and_return(retobj)
+              expect(shell_out_obj).to receive(:__shell_out_command).with(cmd, **options).and_return(retobj)
```

Comment:
This is some fallout from pushing more stuff into instances and having less references to "global" class methods.  Don't think this is a public API.